src/frontend/src/showcase.ts

```diff
@@ -113,58 +114,41 @@ const dummyChallenge: Challenge = {
 const dummyIdentity: IdentifiableIdentity =
   undefined as unknown as IdentifiableIdentity;
 
-const login = (userNumber?: bigint): void => {
-  const returning: Returning =
-    userNumber !== undefined
-      ? {
-          returning: true,
-          clearCache: () => console.log("clear cache"),
-          userNumber: userNumber,
-        }
-      : { returning: false };
-  loginPage({
-    submit: (userNumber) => {
-      console.log("Submitted:", userNumber);
-    },
-    addDevice: (userNumber) => {
-      console.log("Add device:", userNumber);
-    },
-    recover: () => {
-      console.log("Recover");
-    },
-    register: () => {
-      console.log("Register");
-    },
-    ...returning,
+const welcome = () =>
```

Comment:
It would be nice if we had a screenshot / showcase page of the "Authenticate to manage your anchor" page as well.

Also, on that particular page, I'm not sure, labeling the button 'Authorize' makes a lot of sense. I would expect it to be 'Authenticate' or 'Sign in'.